invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -291,6 +291,14 @@
       "title": "List of languages of the document",
       "type": "array",
       "uniqueItems": true
+    },
+    "_items": {
```

Comment:
`eitems`? Or you intend to put different item formats for the document e.g cd, cassette etc.?